Source/ACE.Entity/BaseAceObject.cs

```diff
@@ -10,6 +11,126 @@ public class BaseAceObject
     {
         [DbField("baseAceObjectId", (int)MySqlDbType.UInt32, Update = false, IsCriteria = true)]
         public virtual uint AceObjectId { get; set; }
+        /// <summary>
+        /// TODO: convert to enum
+        /// </summary>
+        public uint ? AmmoType
+        {
+            get
+            {
+                var listItem = AceObjectPropertiesInt.Find(x => x.IntPropertyId == (uint)PropertyInt.AmmoType);
+                if (listItem == null) return null;
```

Comment:
There's this awesome little thing C# did in version 6.  The "?" operator for daisy chaining null checks.  This could be simplified into the following:
return listItem?.PropertyValue;